src/style.css

```diff
@@ -0,0 +1,123 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  height: 406px;
+  width: 198px;
```

Comment:
You can use box-sizing: border-box rule to add border and padding widths to the overall width of the card